spec/features/display_home_page_spec.rb

```diff
@@ -1,24 +1,22 @@
 require 'spec_helper'
 
 describe "Home Page", type: :feature do
-  let(:abby) { FactoryGirl.find_or_create :user_with_fixtures }
-  
-  before do
-    sign_in abby
```

Comment:
The first example, doesn't need a user to be signed in?  Let's do the bare minimal in our setups.